config/webhook/kustomization.yaml

```diff
@@ -1,7 +1,6 @@
 resources:
 - manifests.yaml
 - service.yaml
-- network_policy.yaml
```

Comment:
do we not need this anymore?

Reply:
Added back in - we may need to tweak this to the version you created - this was not generated by operator-sdk